doc/user/content/release-notes.md

```diff
@@ -57,45 +57,133 @@ Wrap your release notes at the 80 character mark.
 
-- Add the advanced [`--timely-progress-mode` and `--differential-idle-merge-effort` command-line arguments](/cli/#dataflow-tuning)
-  to tune dataflow performance.
+{{% version-header v0.6.1 %}}
 
-- Add `ALL` to [`FETCH`](/sql/fetch).
+- Validate `WITH` clauses in [`CREATE SOURCE`](/sql/create-source) and [`CREATE
+  SINK`](/sql/create-sink) statements. Previously Materialize would ignore any
+  invalid options in these statement's `WITH` clauses.
 
-- Change [`FETCH`](/sql/fetch) with no `TIMEOUT` to wait for some rows
-  to be available.
+  Upgrading to v0.6.1 may therefore fail if any of the sources or sinks within
+  have invalid `WITH` options. If this occurs, drop these invalid sources or
+  sinks using v0.6.0 and recreate them with valid `WITH` options.
 
   **Backwards-incompatible change.**
 
-- Consider the following keywords to be fully reserved: `WITH`, `SELECT`,
-  `WHERE`, `GROUP`, `HAVING`, `ORDER`, `LIMIT`, `OFFSET`, `FETCH`, `OPTION`,
-  `UNION`, `EXCEPT`, `INTERSECT`. Previously only the `FROM` keyword was
-  considered fully reserved.
+- Change the default value of the `timeout` option to [`FETCH`](/sql/fetch)
+  from `0s` to `None`. The old default caused `FETCH` to return immediately
+  even if no rows were available. The new default causes `FETCH` to wait for
+  at least one row to be available.
 
-  These keywords can no longer be used as bare identifiers anywhere in a SQL
-  statement, except following an `AS` keyword in a table or column alias. They
-  can continue to be used as identifiers if escaped. See the
-  [Keyword collision](/sql/identifiers#keyword-collision) documentation for
-  details.
+  To maintain the old behavior, explicitly set the timeout to `0s`, as in:
+
+  ```sql
+  FETCH ... WITH (timeout = '0s')
+  ```
 
   **Backwards-incompatible change.**
 
-- Validate `WITH` clauses in `CREATE SOURCE` and `CREATE SINK` statements. This
-  is a potentially backwards-incompatible change as previously any invalid
-  `WITH` clauses would be silently ignored. Upgrading materialize in-place
-  on a persisted catalog that has an invalid `WITH` clause will now fail.
+- Consider the following keywords to be fully reserved in SQL statements:
+  `WITH`, `SELECT`, `WHERE`, `GROUP`, `HAVING`, `ORDER`, `LIMIT`, `OFFSET`,
+  `FETCH`, `OPTION`, `UNION`, `EXCEPT`, `INTERSECT`. Previously only the `FROM`
+  keyword was considered fully reserved.
+
+  You can no longer use these keywords as bare identifiers anywhere in a SQL
+  statement, except following an `AS` keyword in a table or column alias. They
+  can continue to be used as identifiers if escaped. See the [Keyword
+  collision](/sql/identifiers#keyword-collision) documentation for details.
 
   **Backwards-incompatible change.**
 
-- Add `upper` and `lower` to the [string function](/sql/functions#string-func) suite.
+- Change the return type of [`sum`](/sql/functions/#aggregate-func) over
+  [`bigint`](/sql/types/integer)s from `bigint` to
```

Comment:
lets change this to `bigints` instead of `bigint`s

Reply:
In fact I think it's a very widespread typesetting convention!